src/types/spot.ts

```diff
@@ -1929,3 +1924,3114 @@ export interface BSwapLiquidity {
+    free: string;
+    locked: string;
+  }[];
+}
+
+export interface ManagedSubAccountListParams {
+  email?: string;
+  page?: number;
+  limit?: number;
+}
+
+export interface ManagedSubAccountListResponse {
+  total: number;
+  managerSubUserInfoVoList: {
+    rootUserId: number;
+    managersubUserId: number;
+    bindParentUserId: number;
+    email: string;
+    insertTimeStamp: number;
+    bindParentEmail: string;
+    isSubUserEnabled: boolean;
+    isUserActive: boolean;
+    isMarginEnabled: boolean;
+    isFutureEnabled: boolean;
+    isSignedLVTRiskAgreement: boolean;
+  }[];
+}
+
+export interface SubAccountTransactionStatisticsResponse {
+  recent30BtcTotal: string;
+  recent30BtcFuturesTotal: string;
+  recent30BtcMarginTotal: string;
+  recent30BusdTotal: string;
+  recent30BusdFuturesTotal: string;
+  recent30BusdMarginTotal: string;
+  tradeInfoVos: {
+    userId: number;
+    btc: number;
+    btcFutures: number;
+    btcMargin: number;
+    busd: number;
+    busdFutures: number;
+    busdMargin: number;
+    date: number;
+  }[];
+}
+
+export interface ManagedSubAccountDepositAddressParams {
+  email: string;
+  coin: string;
+  network?: string;
+}
+
+export interface ManagedSubAccountDepositAddressResponse {
+  coin: string;
+  address: string;
+  tag: string;
+  url: string;
+}
+
+export interface EnableOptionsForSubAccountResponse {
+  email: string;
+  isEOptionsEnabled: boolean;
+}
+
+export interface ManagedSubAccountTransferTTLogParams {
+  startTime: number;
+  endTime: number;
+  page: number;
+  limit: number;
+  transfers?: string;
+  transferFunctionAccountType?: string;
+}
+
+export interface ManagedSubAccountTransferTTLogResponse {
+  managerSubTransferHistoryVos: {
+    fromEmail: string;
+    fromAccountType: string;
+    toEmail: string;
+    toAccountType: string;
+    asset: string;
+    amount: string;
+    scheduledData: number;
+    createTime: number;
+    status: string;
+    tranId: number;
+  }[];
+  count: number;
+}
+
+export interface UIKlinesParams {
+  symbol: string;
+  interval: string;
+  startTime?: number;
+  endTime?: number;
+  timeZone?: string;
+  limit?: number;
+}
+
+export interface TradingDayTickerParams {
+  symbol?: string;
+  symbols?: string[];
+  timeZone?: string;
+  type?: 'FULL' | 'MINI';
+}
+
+export type TradingDayTickerFullResponse = {
+  symbol: string;
+  priceChange: string;
+  priceChangePercent: string;
+  weightedAvgPrice: string;
+  openPrice: string;
+  highPrice: string;
+  lowPrice: string;
+  lastPrice: string;
+  volume: string;
+  quoteVolume: string;
+  openTime: number;
+  closeTime: number;
+  firstId: number;
+  lastId: number;
+  count: number;
+}[];
+
+export type TradingDayTickerMiniResponse = {
+  symbol: string;
+  openPrice: string;
+  highPrice: string;
+  lowPrice: string;
+  lastPrice: string;
+  volume: string;
+  quoteVolume: string;
+  openTime: number;
+  closeTime: number;
+  firstId: number;
+  lastId: number;
+  count: number;
+}[];
```

Comment:
Same here on hiding the "array" aspect in the type/interface. Better to make this an interface, and where the interface is being used, you would use an array of it.
```suggestion
export interface TradingDayMiniTicker {
  symbol: string;
  openPrice: string;
  highPrice: string;
  lowPrice: string;
  lastPrice: string;
  volume: string;
  quoteVolume: string;
  openTime: number;
  closeTime: number;
  firstId: number;
  lastId: number;
  count: number;
};
```